src/Migration/Logging/SwagMigrationLoggingEntity.php

```diff
@@ -49,13 +49,10 @@ class SwagMigrationLoggingEntity extends Entity
      */
     protected ?array $convertedData = null;
 
-    /**
-     * @var array<string, mixed>|null
-     */
-    protected ?array $usedMapping = null;
-
     protected ?string $exceptionMessage = null;
 
+    protected string $entityId;
```

Comment:
the field is nullable right?
```suggestion
    protected ?string $entityId;
```

Reply:
yes.. in the first step i make it required, but it turned out its not possible